ggshield/utils/git_shell.py

```diff
@@ -95,6 +96,25 @@ def _git_rev_parse(option: str, wd: Path) -> Optional[str]:
     return _git_rev_parse_absolute(option=option, wd_absolute=wd.resolve())
 
 
+def _parse_from_git_url(url: str) -> str:
+    """
+    Removes elements from the git remote url.
+    - scheme
+    - credentials
+    - port
+    - extension
+    https://user:pass@mygitlab.corp.com:84/path/to/repo.git -> mygitlab.corp.com/toto/titi/tata
+    """
+    stripped_scheme = re.sub(r"https?://", "", url)
+    stripped_credentials = re.sub(r".+@", "", stripped_scheme)
+    stripped_port = re.sub(r":\d*/", "/", stripped_credentials)
+    # Github/Gitlab/BitBucket
+    stripped_extension = re.sub(r"\.git$", "", stripped_port)
+    # Azure Devops
+    stripped_extension = re.sub(r"/_git/", "/", stripped_extension)
+    return stripped_extension
```

Comment:
reusing the `url` variable would be simpler here.

Reply:
I was using the variables names as explanations for the regexes.
Is this version better?

Reply:
Looks good, thanks.